tests/page-objects/forms/generic-form.wdio.page.js

```diff
@@ -1,5 +1,7 @@
 const submitButton = () => $('.enketo .submit');
 const nextButton = () => $('button.btn.btn-primary.next-page');
+const nameField = () => $('#report-form form [name="/data/name"]');
+const submittedName = () => $('#reports-content .details ul li:first-child p');
```

Comment:
I don't think we should add this to `genericForm`, especially not under this name. 
This basically selects the first field that is displayed in report details. This is not necessarily a name. 
If you want to include this in `genericForm`, we should rename it to `firstReportDetailField` or similar.